libs/dal/src/repositories/change/change.repository.ts

```diff
@@ -6,6 +6,10 @@ import { ChangeEntity, ChangeDBModel } from './change.entity';
 import { Change } from './change.schema';
 import { UserEntity } from '../user';
 
+type ChangeEntityPopulated = ChangeEntity & {
```

Comment:
Could we export it one level outside 
as we have in environment types? 
libs/dal/src/repositories/environment/types.ts